### PR TITLE
Update package naming rules

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

File: development/extensions/tutorial_basics.rst [hunk 1]
```diff
@@ -38,9 +38,9 @@ The extension name is the name of the extension. In this tutorial we will use ``
 
 .. important::
 
-    Both the vendor and extension names must start with a lower or upper case letter, followed by letters and numbers
-    only. **Underscores, dashes and other characters are not permitted.** It is perfectly fine to have an extension
-    named ``iamanextension``.
+    Both the vendor and extension names must be lowercase and must start with a letter. Names can contain numbers
```
Comment (DavidIQ) [inline development/extensions/tutorial_basics.rst]: "must be lowercase" is incompatible with numbers, which is why the two sentences were joined before. This should probably be:
"...must start with a lowercase letter, followed by lowercase letters and numbers only."

Comment (VSEphpbb) [reply development/extensions/tutorial_basics.rst]: Updated

**Comment by rxu** [comment]
Wasn't it for 3.3/master?

**Comment by VSEphpbb** [comment]
It will be merged up through them too.

**Comment by rxu** [comment]
I mean this have changed extension package naming rules for 3.2.

**Comment by VSEphpbb** [comment]
Yes.

**Comment by VSEphpbb** [comment]
But this is documentation. Not rules ;)

**Comment by VSEphpbb** [comment]
We're no longer going to suggest users do anything but use lowercase to prevent more situations like yours.

**Comment by rxu** [comment]
Okay then. Don't forget to inform Extensions team 😉

**Pull request merged**